internal/snmp/translate.go

```diff
@@ -111,22 +111,29 @@ type MibEntry struct {
 }
 
 func TrapLookup(oid string) (e MibEntry, err error) {
```

Comment:
Wouldn't it be nicer if we just passed an `Oid` type instead of string?
```suggestion
func TrapLookup(givenOid types.Oid) (e MibEntry, err error) {
```

Reply:
The oid arrives from the trap throught gosnmp as a string. If we make TrapLookup take gosmi's types.Oid, we would have to convert from string to types.Oid everywhere TrapLookup (aka lookupFunc) is called. Makes sense to me to leave the argument a string.